fury/tests/test_primitive.py

```diff
@@ -49,6 +49,21 @@ def test_vertices_primitives_octagonalprism():
     npt.assert_equal(vertices.max(), two)
 
 
+def test_vertices_primitives_pentagonalprism():
+    # Testing the default vertices of the primitive pentagonal prism.
+    vertices, _ = fp.prim_pentagonalprism()
+    shape = (10, 3)
+    five = (float('{:.7f}'.format(math.sqrt(5))))
+    onec = (five - 1) / 4.0
+    twoc = (five + 1) / 4.0
+    sone = (math.sqrt(10 + (2 * five))) / 4.0
+    stwo = (math.sqrt(10 - (2 * five))) / 4.0
+    npt.assert_equal(vertices.shape, shape)
+    npt.assert_equal(np.mean(vertices), 0)
```

Comment:
this test is failing because your primitive is not centered.

Reply:
Can you please help me to get it centered?
How can I do?

Reply:
Please, provide a screenshot of your actor with axes actor and then, we can have an idea and help you

Reply:
Done!

Reply:
Please provide a better visual where the axes are visible, see [this](https://github.com/fury-gl/fury/pull/393) for reference.